src/app/features/todos/todo-details/todo-details.component.html

```diff
@@ -44,6 +30,7 @@ <h1 class="flex-grow">
     *ngIf="!entity.isNew"
     mat-icon-button
     [matMenuTriggerFor]="additional"
+    style="margin-top: -8px"
```

Comment:
This was necessary to not have an outer scroll bar when the buttons wrap into 2 rows. Probably because the icon button is too tall.

Reply:
```suggestion
    style="margin-top: -8px; margin-bottom: -8px"
```